data-raw/iran_mapping.R

```diff
@@ -0,0 +1,67 @@
+library(data.table)
```

Comment:
Does this need any references?

Reply:
Maybe a GBD 2017 reference. I'm not really sure who made it internally